packages/react-router/src/router.ts

```diff
@@ -853,6 +858,12 @@ export class Router<
         },
       })
     }
+
+    if (!this.isServer) {
+      this.isViewTransitionTypesSupported = CSS.supports(
```

Comment:
Looking at the failures with the unit tests, we probably should perform a check for `window` or something like `typeof window !== 'undefined' && 'CSS' in window` and then do the check as `window.CSS.supports(...)`.

Reply:
Should I still include `!this.isServer` alongside these new checks?